posthog/api/authentication.py

```diff
@@ -258,9 +259,20 @@ def create(self, validated_data):
             user = None
 
         if user:
```

Comment:
if there's no specific business logic here with throttling, I'd recommend using the DRF throttling class instead: https://www.django-rest-framework.org/api-guide/throttling/

For an example, we use it elsewhere in app to rate limit other requests: https://github.com/PostHog/posthog/blob/master/posthog/rate_limit.py#L133

Nicer this way since the entire app uses a consistent method for rate limiting things.

Reply:
That is much simpler!